prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -141,6 +145,11 @@ elements:
   - name: standard.testing_lab_county
     csvFields: [{name: Testing_lab_county}]
 
+  - name: organization_name
```

Comment:
This was a request from SimpleReport - they want to break apart 'facility' and 'organization', so we added this new field.  Facility still uses 'ordering_facility_name'.